components/pages/WarningPage/WarningPage.module.css

```
@@ -9,6 +9,28 @@
   align-items: center;
   gap: 80px;
 }
+@media screen and (max-width: 720px) {
+  .contentSection {
+    flex-direction: column;
+    margin: 0 auto;
+    gap: 40px;
+  }
+  .contentSection svg {
+    width: 150px;
+    height: 150px;
+  }
+  .textContainer {
+    max-width: 100%;
+    transform: scale(0.85);
```

This looks a bit like a hack. I think you need this instead:
```suggestion
    margin: 20px;
```